autogtp/Management.cpp

```diff
@@ -57,8 +57,36 @@ Management::Management(const int gpus,
     m_fallBack(Order::Error) {
 }
 
+void Management::runTuningProcess(const QString &tuneCmdLine) {
+    QTextStream(stdout) << tuneCmdLine << endl;
+    QProcess tuneProcess;
+    tuneProcess.start(tuneCmdLine);
```

Comment:
If the user is running a CPU only client, it will reject --tune-only, so you need to handle this case and just bail out.

Reply:
But the process will run, say something like "no OpenCl capable Device found" and exit. The code should not have any problem with this.